test.py

```diff
@@ -0,0 +1,12 @@
+import gatoryaml
```

Comment:
Just make this a test inside the `tests/` folder, if you want to have a full integration test. Otherwise, I would try to keep manual testing out of the repository, especially the root directory.